src/Controller/Admin/ModuleConfiguration.php

```diff
@@ -36,7 +38,11 @@ public function render()
     public function saveConfVars()
     {
         if ($this->isFactFinder()) {
-            $_POST['confaarrs']                       = array_reduce($this->localizedFields, [$this, 'fromArray'], $_POST['confaarrs'] ?? []);
+            $_POST['confaarrs'] = array_reduce($this->localizedFields, function (array $result, string $field): array {
```

Comment:
I found that Oxid has `getRequestEscapedParameter` method in `OxidEsales\EshopCommunity\Core\Request` class, can we use it instead of `$_POST`/`$_GET`?

Reply:
In case of setting - i don't think so
In case of getting - yeah, probably. I will check this